deploy/kubernetes/google_nvme_id

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
```

Comment:
Understood thanks, are these files going to be added in the image? I didn't see any changes to the Dockefile files

Reply:
@lizhuqi I think it would be fine to add the dockerfile change to this PR too unless you think there's a reason to keep it separate? Having it in the same change does self-document how these are used.

Reply:
oh I prefer to have another PR for the dockerfile change.

Reply:
I am also fine to add the dockerfile change as well. I will add the dockerfile change to this PR.

Reply:
👍🏾